src/components/Input/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { forwardRef } from "react";
+import useIsDeprecatedForm from "../../hooks/internal/useIsDeprecatedForm";
+import { ComponentWithProperties } from "../../types";
+import { ComponentName } from "../componentName";
+import { default as InputInternal } from "./Input";
+import { default as InputDeprecated } from "./Input_deprecated";
+import {
+  InputProps,
+  setDeprecatedProps as shouldUseDeprecatedInput,
+} from "./types";
+
+const { TYPES, VARIANTS, COLORS, DEFAULT_PROPS } = InputDeprecated;
+
+interface StaticProperties {
+  displayName: string,
+  TYPES: string[],
+  VARIANTS: string[];
+  COLORS: string[];
+  DEFAULT_PROPS: typeof InputDeprecated.DEFAULT_PROPS;
+}
+
+const Input = forwardRef(function InputComponent(props: InputProps, ref) {
```

Comment:
Can this be an arrow func ? 
```
(props: InputProps, ref) => { ...
```

Reply:
It can't be an arrow function due, have added a comment on the line above to the reason